docs/source-pytorch/extensions/loops.rst

```diff
@@ -150,12 +149,7 @@ Here is a simple example how to add a new hook:
 
     class CustomFitLoop(FitLoop):
         def advance(self):
-            # ... whatever code before
-
-            # pass anything you want to the hook
-            self.trainer.call_hook("my_new_hook", *args, **kwargs)
```

Comment:
The `Trainer.call_hook` was still used in this part of the documentation. I am unsure whether any additional documentation should be written for `call_callback_hooks`, `call_lightning_module_hook`, `call_accelerator_hook` and `call_ttp_hook`? For now, I just removed the part referencing `call_hook`. Should I file an issue on additional information on how to use these hooks, or does it already exist?

Reply:
@awaelchli

Reply:
It's ok to remove it.